phac-nml/irida-next
# Namespace Attachments: Upload and Destroy Dialogs to Components

## What does this PR do and why?
This PR migrates the namespace attachment upload and destroy dialogs to a component.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
In both projects and groups:
1. Test uploading and destroying attachments. Ensure the dialog text/UI are as expected and attachments still upload and delete properly

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
